build.py

```diff
@@ -668,6 +684,7 @@ def create_dockerfile_linux(ddir, dockerfile_name, argmap, backends, repoagents,
 # so explicit set on all of include
 RUN chown -R triton-server:triton-server include
 '''
+    df += install_dcgm_libraries()
```

Comment:
This should be up at line 659 where the other common dependencies are installed.